reciprocalspaceship/dataset.py

```diff
@@ -539,16 +548,21 @@ def hkl_to_observed(self, m_isym="M/ISYM", inplace=False):
             dataset = self.copy()
 
         hkls = dataset.get_hkls()
-        isym = (dataset[m_isym] % 256).to_numpy() # Convert M/ISYM to ISYM
-        observed_hkls = hkl_to_observed(hkls, isym, dataset.spacegroup)
 
-        # Update HKLs
+        # GH#3: Separate combined M/ISYM into M and ISYM
+        isym = (dataset[m_isym] % 256).to_numpy()
```

Comment:
can we change the default value of `m_isym` to `None`, and check the `self.dtypes` for any `rs.dtypes.m_isym`? we will need to raise some kind of error in the case that multiple columns have `rs.dtypes.m_isym`.

Reply:
I agree. This will be handled with an analogous method to `DataSet.get_phase_keys()`.